pkg/proto/configuration/grpc/grpc.proto

```diff
@@ -501,3 +511,15 @@ message TracingMethodConfiguration {
   // 'attributes_from_first_request_message'.
   repeated string attributes_from_first_response_message = 2;
 }
+
+message ServerRelayConfiguration {
+  // The remote gRPC server to forward the gRPC calls to.
+  ClientConfiguration endpoint = 1;
+
+  // The full gRPC service and method name to relay.
+  // Examples of valid names include:
+  //
+  // /build.bazel.remote.execution.v2.Execution/Execute
+  // /com.google.devtools.build.v1.PublishBuildEvent/PublishBuildEvent
+  repeated string methods = 2;
```

Comment:
Is it really preferable to do this on a per method basis? Wouldn't it be friendlier to do this per service?